pkg/generator/testfiles/cloud-init

```diff
@@ -6,6 +6,8 @@ YmFy
 EOF
 chmod '0600' '/foo'
 
+
```

Comment:
```suggestion
```
Remove the lines or do they serve an other purpose?

Reply:
The generated file will differ from the test file and the test will fail, if I remove the extra empty lines.